mdk/build.gradle

```diff
@@ -209,7 +209,6 @@ jar {
                 "Implementation-Title"    : project.name,
                 "Implementation-Version"  : project.jar.archiveVersion,
                 "Implementation-Vendor"   : mod_authors,
-                "Implementation-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ")
```

Comment:
This is a useful field during debugging.
And MANIFEST.MF files should be skipped during binary comparison checks anyway due to signature content which is not remotely reproducible. So whether this field exists or not is irrelevant.